vault/core.go

```diff
@@ -2183,6 +2183,9 @@ func (c *Core) preSeal() error {
 	if err := c.unloadMounts(context.Background()); err != nil {
 		result = multierror.Append(result, fmt.Errorf("error unloading mounts: %w", err))
 	}
+
+	atomic.StoreUint64(c.counters.requests, 0)
```

Comment:
I would say, at a minimum, this should have some subset of the PR comment explaining why we're going to do this?

Reply:
roger, can do that!

Reply:
968ba28

Reply:
so has `c.counters.requests` already been persisted by this point? or, how does that stuff work?

Reply:
Yep, so `c.counter.requests` is being persisted every syncinternval here https://github.com/hashicorp/vault/blob/main/vault/counters.go#L145 by the active node ( and synced to the active if a standby ). The default on syncInterval is every 30 seconds. This is the block where this logic happens https://github.com/hashicorp/vault/blob/main/vault/core_metrics.go#L101-L126